spec/inject_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'inject'
+
+describe Array do
+  describe 'new_inject_block' do
+    it 'should use first element of array as default argument' do
+      expect([3,4,5].new_inject_block {|a,b| a + b}).to eq 12
```

Comment:
Space missing after comma.<br>Space between { and | missing.<br>Space missing inside }.